internal/namespaces/instance/v1/custom_server_create_test.go

```diff
@@ -230,6 +230,7 @@ func Test_CreateServer(t *testing.T) {
 }
 
 // None of the tests below should succeed to create an instance.
+// these tests need to be run in sequence
```

Comment:
Why ? 

Usually, comments should explain `why`, not `what`.

Reply:
Good question :sweat_smile: it doesnt work in parallel but works in sequence. Not sure why :thinking:

Reply:
@loicbourgois fixed